### ISSUE TITLE
Follow up [2022 Feb] on decoupling of Transifex account between edX and Open edX

### ISSUE THREAD

**Issue by kdmccormick** [openedx/tcril-engineering#82]
This (private) [issue from the decoupling board](https://github.com/openedx/decoupling/issues/12) has a goal of:
* transferring the existing Open edX Transifex account to tCRIL, and
* having edX migrate private projects out to their own Transifex account.

That work is in edX's ballpark. In mid-February 2022 we should follow up with them.

**Comment by kdmccormick** [comment]
@brian-smith-tcril , do you think this is still relevant? (let me know if you can't access that private link)

**Comment by brian-smith-tcril** [comment]
I think this is still relevant, the `open-edx` transifex account currently has the following projects

<img width="322" height="1047" alt="Image" src="https://github.com/user-attachments/assets/696945d8-547a-4e0e-b0d6-22eac793861d" />

but I think we really should only have the `openedx-translations*` ones now.

**Comment by sarina** [comment]
@brian-smith-tcril what would be the next steps here?

**Comment by brian-smith-tcril** [comment]
I think we need to figure out what would happen if we deleted all the projects that *aren't* `openedx-translations*`

**Comment by sarina** [comment]
@brian-smith-tcril do you know if 2U has their own transifex account, and if they could transfer those resources with their translations into their own account?

**Comment by brian-smith-tcril** [comment]
I do not know if 2U has their own transifex account or not. I think they might but I'm not sure who to ask to confirm.

**Comment by sarina** [comment]
Before I reach out and ask - is Prospectus the only resource that belongs to 2U? It looks like it's archived - https://app.transifex.com/open-edx/prospectus/

In fact it appears that _most_ of our projects are archived, except the `openedx-translations*` - https://app.transifex.com/open-edx/settings/archive/ - I am going to look into the DemoX translations, I don't think we have an integration set up for the new Demo course so that might be archivable as well. I think in this light, we can close this issue- agree?

**Comment by sarina** [comment]
Yeah... I archived the DemoX course, it's using a way-outdated version of that course.
